VideoLocker/res/values/styles.xml

```diff
@@ -306,23 +307,24 @@
         <item name="android:paddingTop">@dimen/vertical_input_padding</item>
     </style>
 
-    <style name="edX.Widget.Button" parent="@android:style/Widget.TextView">
```

Comment:
This will introduce some issues e.g. button has a default background, which will be added to the Buttons we are currently using in app while and some gravity issues.
More discussion here:
https://github.com/edx/edx-app-android/pull/371#discussion_r40097031

Issue:
![navigation](https://cloud.githubusercontent.com/assets/1710804/11897852/ebd280a2-a5b4-11e5-8b74-b21770447021.png)